settings.py

```diff
@@ -1,12 +1,3 @@
-#############################################
```

Comment:
Why is this part removed?

It is intended to prevent people from trying to write in their secret values in this file, and the later accidentally exposing them when they commit the code.

Reply:
Ah - that makes sense. I was thinking that it was strange that I needed to edit a file that said it shouldn't be edited. I assumed that the comment was supposed to be in config-template.py, and was warning that editing that file wouldn't actually cause changes that would be picked up, since it looks for config.py, which, being a file not published in the repo, isn't intuitively one people would find. 

Maybe instead of saying "Don't edit this file" it should say "Don't add client secrets directly to this file"?

Reply:
👍